docs/roadmap.md

```diff
@@ -1,107 +1,112 @@
 # Amundsen Roadmap
+
 The following roadmap gives an overview of what we are currently working on and what we want to tackle next. This helps potential contributors understand the current status of your project and where it's going next, as well as giving a chance to be part of the planning.
 
 ## Amundsen Mission
-> *To organize all information about data and make it universally actionable*
 
-## Vision for 2020
-> *Centralize a comprehensive and actionable map of all our data resources that can be leveraged to solve a growing number of use cases and workflows*
+> _To organize all information about data and make it universally actionable_
 
-## Short Term - Our Current focus
+## Vision for 2021
 
-#### Provide Rich metadata to make data trust worthy
-*What*: Enrich table detail page with additional structure metadata / programmatic description.
+> _Centralize a comprehensive and actionable map of all our data resources that can be leveraged to solve a growing number of use cases and workflows_
 
-*Status*: tech spec WIP
+## Short Term - Our Current focus
 
 #### Native lineage integration
-*What*: We want to create a native lineage integration in Amundsen, to better surface how data assets interact with each other.
 
-*Status*: tech spec out
+_What_: We want to create a native lineage integration in Amundsen, to better surface how data assets interact with each other.
```

Comment:
if we are only targeting table level lineage in short term, we may want to make it explict.

Reply:
I think we want both, and we have designs for both